rust/src/lib.rs

```diff
@@ -11,22 +11,19 @@ pub fn base64_to_hex(s: &str) -> String {
     let bytes = base64::decode(s).unwrap();
     hex::encode(&bytes)
 }
-//
-//pub fn xor_hex(hex1: &str, hex2: &str) -> String {
-//    if hex1.len() != hex2.len() { }
-//    let bytes =
-//}
 
+pub fn num_blocks(key: &[u8], text: &[u8]) -> usize {
+    let mut num_blocks = text.len() / key.len();
```

Comment:
completely optional, but I personally like reducing mutable variables if possible. This could be re-written as
```rust
let num_blocks = text.len() / key.len();
    if text.len() % key.len() > 0 {
        num_blocks + 1
    }else{
        num_blocks
    }
```

Reply:
oh that makes sense